magento2/magento2/scripts/init-and-run.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -x
+
+# Turn on monitor mode, required by job control
+set -m
+
+if [ ! -f /var/www/html/magento2/.installed ]; then
+  cd /var/www/html/magento2/ || exit
+  composer config --global repos.packagist composer https://packagist.org
+  composer require nosto/php-sdk:@stable
+
+  until mysql -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" -h"${MYSQL_HOST}"; do
+    >&2 echo "MySQL is unavailable - sleeping"
+    sleep 5
+  done
+
+  until curl -i ${ES_HOST}:9200/_cluster/health; do
```

Comment:
This should be done in the docker-compose using a health-check https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y